docs/infrastructure-components.md

```diff
@@ -0,0 +1,164 @@
+# Infrastructure components
+
+We describe here the elements that form the infrastructure required for projects
+under KubeVirt's umbrella. This mainly involves components required to run our
+CI/CD systems and related services, like the monitoring stack or applications for
+indexing and querying build logs.
+
+The CI/CD system is based on [Prow], kubernetes CI system. For a brief overview
+of how Prow runs jobs take a look at ["Life of a Prow Job"]. To see common Prow
+usage and interactions flow, see the pull request interactions [sequence diagram].
+
+## Layout
+
+![infra-layout](infra-layout.png)
+
+## Clusters
+
+Our infrastructure currently uses 3 main clusters. For each of them we will
```

Comment:
Do we have non-main clusters? Shouldn't we list them all here?

How about
```suggestion
Our infrastructure includes the following clusters. For each of them, this document provides the following fields
```

Reply:
Sounds good :+1: